src/error_messages/error_messages.h

```diff
@@ -489,6 +489,12 @@
 #define FIM_DB_ERROR_RM_NOT_SCANNED                 "(6709): Failed to delete from db all unscanned files."
 #define FIM_ERROR_WHODATA_INIT                      "(6710): Failed to start the Whodata engine. Directories/files will be monitored in Realtime mode"
 
+/* Wazuh Logtest error messsages */
+#define LOGTEST_ERROR_BIND_SOCK                     "(7100): At wazuh_logtest_init(): Unable to bind to socket '%s'. Errno: (%d) %s"
```

Comment:
Please remove "At *():" as the logging library will add it automatically if debugging is enabled.